VirtoCommerce.Platform.Web/Controllers/Api/LicensingController.cs

```diff
@@ -0,0 +1,94 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Hosting.Internal;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Extensions;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Net.Http.Headers;
+using VirtoCommerce.Platform.Core.Security;
+using VirtoCommerce.Platform.Web.Extensions;
+using VirtoCommerce.Platform.Web.Licensing;
+
+namespace VirtoCommerce.Platform.Web.Controllers.Api
+{
+    [Produces("application/json")]
+    [Route("api/platform/licensing")]
+    //[Authorize(SecurityConstants.Permissions.ModuleManage)]
+    [ApiExplorerSettings(IgnoreApi = true)]
+    public class LicensingController : Controller
+    {
+        private readonly IHostingEnvironment _hostingEnv;
+        public LicensingController(IHostingEnvironment hostingEnv)
+        {
+            _hostingEnv = hostingEnv;
+        }
+
+        [HttpPost]
+        [Route("activateByCode")]
+        [ProducesResponseType(typeof(License), 200)]
+        public async Task<IActionResult> ActivateByCode([FromBody]string activationCode)
+        {
+            License license = null;
+
+            using (var httpClient = new HttpClient())
+            {
+                var activationUrl = new Uri("https://virtocommerce.com/admin/api/licenses/activate/" + activationCode);
```

Comment:
need to rename VirtoCommerce.Platform.Web.Infrastructure.DemoOptions type  to  -> PlatformOptions
and add to it the "LicenceActivationUrl" property. And use it value from it instead of hardcoded value